chainerx_cc/chainerx/cuda/cuda_device/comparison.cu

```diff
@@ -135,6 +136,85 @@ public:
 
 CHAINERX_REGISTER_OP_CUDA(LogicalNotOp, CudaLogicalNotOp);
 
+template <typename In, typename Out>
```

Comment:
`Out` does not need to parameterized because it is always `bool` type.

Reply:
Oops should have noticed that. Thank You.